Problem building on arch linux using genie generated Makefile

Hi, I followed the instructions and am getting this error, any idea what could be the cause?

```sh
==== Building SoloudDynamic (debug) ====
Creating debug/Debug/SoloudDynamic
Creating debug/Debug/SoloudDynamic/src/c_api
soloud_c.cpp
Linking SoloudDynamic
/usr/bin/ld: ../../lib/libsoloud_x86_d.a(soloud_sfxr.o): relocation R_X86_64_32 against `_ZTVN6SoLoud12SfxrInstanceE' can not be used when making a shared object; recompile with -fPIC
../../lib/libsoloud_x86_d.a: error adding symbols: Bad value
collect2: error: ld returned 1 exit status
SoloudDynamic.make:104: recipe for target '../../lib/libsoloud_x86_d.so' failed
make[1]: *** [../../lib/libsoloud_x86_d.so] Error 1
Makefile:40: recipe for target 'SoloudDynamic' failed
make: *** [SoloudDynamic] Error 2
```

--- a/build/genie.lua
+++ b/build/genie.lua
@@ -328,7 +328,10 @@ solution "SoLoud"
 	--       doesn't do this well on it's own and is recommended to setup this
 	--       manually. See https://github.com/bkaradzic/bx/blob/master/scripts/toolchain.lua
 	configuration { "gmake" }
-		buildoptions { "-msse4.1" }
+		buildoptions { 
+			"-msse4.1", 
+			"-fPIC"
+		}
 
     configuration {}
 

